### DISCUSSION TITLE
Flowgorithm file to PDF

### DISCUSSION THREAD

**Discussion by ShuttlezDev** [nigification/Flowgorithm-Mac-OSX#13] [Q&A] [answered]
I've been unsucessful in finding a way to turn my flowgorithm file to a PDF. It won't let me save it as any other file on Mac, but I know Windwos has a Print program that automatically changes it. How can I change it to PDF?

**Comment by jostasik** [comment, marked as answer]
If you press the key combination "Command + P" this dialog should appear!

![image](https://github.com/jostasik/Flowgorithm-macOS/assets/31175268/36bc1047-4e31-4b82-9a26-e860a324c7eb)
